tests/wpt/mozilla/tests/css/canvas_without_context.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
```

Comment:
This should be a [reference test](http://testthewebforward.org/docs/reftests.html) to verify that the rendered output matches what we expect.